### PR TITLE
fix: updated netty dependancies, fixed illegal netty internal usage

### PR DESCRIPTION
**Pull request by GregoryIvo**
updated netty in opcua driver and target platform to mitigate CVE CVE-2022-41915.
The vulnerability is classified with a CVSS score of 8.2 (High)

**Related Issue:** n/a

**Description of the solution adopted:** Upgraded`io.netty:netty-codec` to version 4.1.86.Final as the next version higher is not final.

**Screenshots:** If applicable, add screenshots to help explain your solution

**Any side note on the changes made:** n/a

### CODE REVIEW

**Comment by GregoryIvo** [comment]
Something seems strange here. I can't seem to figure out these new errors 🤔

**Comment by salvatore-coppola** [comment]
@GregoryIvo There are some tests failing, have you checked [here ](https://ci.eclipse.org/kura/job/multibranch/job/PR-4321/5/testReport/) ?

File: target-platform/p2-repo-common/pom.xml [hunk 1]
```diff
@@ -442,6 +442,11 @@
                                     <artifactId>artemis-native</artifactId>
                                     <version>${org.apache.activemq.artemis.upstream.version}</version>
                                 </dependency>
+                                <artifactItem>
+                                    <groupId>io.netty</groupId>
+                                    <artifactId>netty-all</artifactId>
+                                    <version>${io.netty.version}</version>
```
Comment (salvatore-coppola) [inline target-platform/p2-repo-common/pom.xml]: Doesn't Netty-all somehow conflict with other explicit dependencies?

**Review by salvatore-coppola — APPROVED** [review]

**Comment by GregoryIvo** [comment]
Note: we still need to get QA on this.

**Comment by MMaiero** [comment]
@GregoryIvo Did you reach out to Marko or Nemanja to perform a round of QA?

**Comment by GregoryIvo** [comment]
This PR has passed QA can be merged

**Pull request merged**